pkgs/tools/text/fastmod/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fastmod";
+  version = "v0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "facebookincubator";
+    repo = pname;
+    rev = version;
+    sha256 = "0089a17h0wgan3fs6x1la35lzjs1pib7p81wqkh3zcwvx8ffa8z8";
+  };
+
+  cargoSha256 = "0i5i9hsqynb7c1wv63p7y9k7yfl7iz67favrml97gy8l0dzi0666";
+
+  buildInputs = (stdenv.lib.optional stdenv.isDarwin Security);
+
+  meta = with stdenv.lib; {
+    description = "A utility that makes sweeping changes to large, shared code bases";
+    homepage = "https://github.com/facebookincubator/fastmod";
+    license = "Apache-2.0";
+    maintainers = with maintainers; [ swolchok ];
```

Comment:
Add a separate commit (ordered before the commit that adds this derivation) to add yourself to `maintainers/maintainer-list.nix`.

Reply:
@danieldk thanks for your quick response and suggestions. I've applied them and added myself as a maintainer. All CI checks passed. PTAL.

Reply:
Thanks for making these changes! Could you change the first line of the message of the commit where you add yourself to `maintainers.nix` to:

```
maintainers: add jduan
```

And then squash all the other commits (the ones for the derivation itself) into one commit with the following initial line in the commit message:

```
fastmod: init at 0.4.0
```

All commit messages in `nixpkgs` follow the same patterns to make it easy to find what was changed when ;).

In the commit history, the commit that adds you to `maintainers.nix` should be ordered before the one where the derivation is added.

If you need help fixing up the history, you can enable *Allow edits and access to secrets by maintainers* on in the rightmost column and let me know, and I can do the cleanup.

Reply:
@danieldk thanks for suggesting the convention used by this repo! I've changed the commit history as you suggested. PTAL. Again, thanks for making my first contribution to this repo a pleasant experience!